web/admin/application/configs/klear/model/NotificationTemplates.yaml

```diff
@@ -29,7 +29,7 @@ production:
           'voicemail': ngettext('Voicemail', 'Voicemails', 1)
           'fax': ngettext('Fax', 'Faxes', 1)
           'lowbalance': _('Low balance')
-#          'limit': ngettext('Limit', 'Limits', 1)
+          'invoice': _('Invoice')
```

Comment:
you can use `ngettext('Invoice', 'Invoices' 1)` if you want to reuse translations